backend/usb-libusb.c

```diff
@@ -1597,7 +1606,10 @@ open_device(usb_printer_t *printer,	/* I - Printer */
   }
   number1 = confptr->bConfigurationValue;
 
-  if (number1 != current)
+  if (printer->quirks & USB_QUIRK_NO_ALT_SET)
```

Comment:
Same here, we don't need to know that we are skipping, just whether we did the alt set...